packages/ui5-proxy-middleware/src/base/utils.ts

```diff
@@ -185,7 +185,11 @@ export const setHtmlResponse = (res: any, html: string): void => {
         res.write(html);
         res.end();
     } else {
-        res.status(200).contentType('html').send(html);
+        res.writeHead(200, {
```

Comment:
With this change, do we still need the if or could we also execute `res.writeHead(200, { ...` in a liveload scenario?

Reply:
You can also combine both scenarios - I wasn't sure about the content-type which is not set in the livereload scenario - that's why I kept it as is...

Reply:
Ok, let's leave it as-is for now.